source/_components/weather.buienradar.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: "Buienradar Weather"
+description: "Instructions how to integrate buienradar.nl weather within Home Assistant."
+date: 2017-05-15 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: 
+ha_category: Weather
+ha_release: 0.44
+---
+
+The `buienradar` platform uses [buienradar.nl](http://buienradar.nl/) as an source for current meteorological data for your location. The 
+weather forecast is delivered by Buienradar, who provides a webservice that provides detailed weather information for users in The Netherlands.
+The relevant weatherstation used will be automatically selected based on the location specified in the HA config (or in the buienradar weather/sensor component).
+
+To add the buienradar sensor to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+weather:
+  - platform: buienradar
+    name: buienradar
```

Comment:
Remove the `name` variable since it's not a required variable. It's better to limit the example configuration to only required variables since it's often copy and pasted.